simulation/src/range_likelihood.cpp

```diff
@@ -678,101 +865,96 @@ pcl::simulation::RangeLikelihood::getPointCloud (pcl::PointCloud<pcl::PointXYZRG
       {
-        float z = zf*zn/((zf-zn)*(d - zf/(zf-zn)));
+        float z = zf * zn / ((zf - zn) * (d - zf / (zf - zn)));
 
         // TODO: add mode to ignore points with no return i.e. depth_buffer_ ==1
-        // NB: OpenGL uses a Right Hand system with +X right, +Y up, +Z back out of the screen,
-        // The Z-buffer is natively -1 (far) to 1 (near)
-        // But in this class we invert this to be 0 (near, 0.7m) and 1 (far, 20m)
+        // NB: OpenGL uses a Right Hand system with +X right, +Y up, +Z back out of the
+        // screen, The Z-buffer is natively -1 (far) to 1 (near) But in this class we
+        // invert this to be 0 (near, 0.7m) and 1 (far, 20m)
         // ... so by negating y we get to a right-hand computer vision system
         // which is also used by PCL and OpenNi
         pc->points[idx].z = z;
-        pc->points[idx].x = (static_cast<float> (x)-camera_cx_) * z * (-camera_fx_reciprocal_);
-        pc->points[idx].y = (static_cast<float> (y)-camera_cy_) * z * (-camera_fy_reciprocal_);
-
-        int rgb_idx = y*col_width_ + x;  //camera_width_
-        pc->points[idx].b = color_buffer[rgb_idx*3+2]; // blue
-        pc->points[idx].g = color_buffer[rgb_idx*3+1]; // green
-        pc->points[idx].r = color_buffer[rgb_idx*3]; // red
+        pc->points[idx].x =
+            (static_cast<float>(x) - camera_cx_) * z * (-camera_fx_reciprocal_);
+        pc->points[idx].y =
+            (static_cast<float>(y) - camera_cy_) * z * (-camera_fy_reciprocal_);
+
+        int rgb_idx = y * col_width_ + x;                  // camera_width_
+        pc->points[idx].b = color_buffer[rgb_idx * 3 + 2]; // blue
+        pc->points[idx].g = color_buffer[rgb_idx * 3 + 1]; // green
+        pc->points[idx].r = color_buffer[rgb_idx * 3];     // red
         points_added++;
       }
-      else if (organized)
-      {
+      else if (organized) {
         pc->is_dense = false;
-        pc->points[idx].z = std::numeric_limits<float>::quiet_NaN ();
-        pc->points[idx].x = std::numeric_limits<float>::quiet_NaN ();
-        pc->points[idx].y = std::numeric_limits<float>::quiet_NaN ();
+        pc->points[idx].z = std::numeric_limits<float>::quiet_NaN();
+        pc->points[idx].x = std::numeric_limits<float>::quiet_NaN();
+        pc->points[idx].y = std::numeric_limits<float>::quiet_NaN();
         pc->points[idx].rgba = 0;
       }
     }
   }
 
-  if (!organized)
-  {
-    pc->width    = 1;
-    pc->height   = points_added;
-    pc->points.resize (points_added);
+  if (!organized) {
+    pc->width = 1;
+    pc->height = points_added;
+    pc->points.resize(points_added);
   }
 
-  if (make_global)
-  {
+  if (make_global) {
     // Go from OpenGL to (Z-up, X-forward, Y-left)
     Eigen::Matrix4f T;
-    T <<  0, 0, -1, 0,
-         -1, 0,  0, 0,
-          0, 1,  0, 0,
-          0, 0,  0, 1;
-    Eigen::Matrix4f m = pose.matrix ().cast<float> ();
+    T << 0, 0, -1, 0, -1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1;
```

Comment:
Do we want to preserve these matrix shapes? This one isn't special but the next one in L746 (original) looks sorta neat